CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+- Added unique email overlay
```

Comment:
If possible you should provide corresponding issue numbers to the changelog when it is appropriate.

The change that introduces this PR for issue #8 may affect users directly. I would add some additional explanation of what to expect from this change. In my eyes that is basically a behavioural change.

```suggestion
### Changed
- Change user's email address to be unique in the LDAP directory
   - existing user data are kept without change, even those with non-unique email addresses
   - updating a person's directory entry will lead to an error `some attributes not unique` if the (non-unique) email address is supposed to be kept
```

Reply:
Looks good to me. Commited this